docs/azure/aksextensions.md

```diff
@@ -0,0 +1,129 @@
+---
+Order: 7
+Area: azure
+TOCTitle: AKS Extensions
+ContentId: 131f9633-5446-4384-96ca-7bff2e3dc0fc
+PageTitle: Working with AKS tools and diagnostics in Visual Studio Code
+DateApproved: 5/03/2022
+MetaDescription: Working with AKS tools and diagnostics in Visual Studio Code
+---
+# Working with AKS tools and diagnostics in VS Code
+
+This document will walk you through some of the ways you can interact with your AKS clusters directly from Visual Studio Code. [Azure Kubernetes Services](https://azure.microsoft.com/en-in/services/kubernetes-service/#overview) is a fully managed [Kubernetes](https://azure.microsoft.com/en-in/topic/what-is-kubernetes/#overview) service. Azure Kubernetes Service (AKS) offers serverless Kubernetes, an integrated continuous integration and continuous delivery (CI/CD) experience and enterprise-grade security and governance, is an open-source system for automating deployment, scaling, and management of containerized applications. We will show you how to run [diagnostic health-checks](https://docs.microsoft.com/en-gb/azure/aks/concepts-diagnostics) on your AKS cluster, launch [AKS Periscope](https://github.com/azure/aks-periscope) for more in-depth troubleshooting, deploy [Azure Service Operator](https://github.com/Azure/azure-service-operator), or generate [GitHub Actions Starter Workflows](https://github.com/actions/starter-workflows).
+
+## Before you begin
+
+The [AKS Visual Code Extension](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) has parent dependency on [Kubernetes Visual Code extension](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools). You will need to download the [kubernetes VS Code extension](https://code.visualstudio.com/docs/azure/kubernetes). You can follow this guide on an existing AKS cluster or you can [create AKS cluster](https://docs.microsoft.com/en-us/azure/aks/learn/quick-kubernetes-deploy-portal).
+
+## Install the Azure Kubernetes Services extension
+
+For a fully integrated Azure Kubernetes diagnostics experience, you can install the [Azure Kubernetes Services Tools](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) extension.
+
+With the extension you can easily view and manage your AKS clusters from your Visual Studio Code environment.
+
+To install the Azure Kuberentes Services Visual Code extension, open the Extensions view (`kb(workbench.view.extensions)`) and search for "AKS". Select the Microsoft [Azure Kuberentes Services](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) extension.
```

Comment:
"Visual Studio Code extension"
You can also use just "VS Code" subsequent mentions after the first mention of "Visual Studio Code" at the top of the article. A little easier to read.